sound.js

```diff
@@ -26,7 +26,32 @@ function Sound(filename, basePath, onError, options) {
     }
   }
 
+  this.registerOnPlay = function() {
+    if (this.onPlaySubscription != null) {
+      console.warn('On Play change event listener is already registered');
+      return;
+    }
+
+    if (!IsWindows) {
+      this.onPlaySubscription = eventEmitter.addListener(
+        'onPlayChange',
+        (param) => {
+          const { isPlaying, key } = param;
+          if (key == this._key) {
```

Comment:
- Please use `==` over `===`. I think the only case we need `!=` or `==` is `if (value != null)` to check both `undefined` and `null`. Other cases are little complicated so, not recommended.
- I also found that `===` is faster than `==` (https://stackoverflow.com/a/19042070/1609620).